extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -536,28 +559,77 @@ export class AmpStoryEmbeddedComponent {
           const targetRect = target./*OK*/getBoundingClientRect();
           const pageRect = this.componentPage_./*OK*/getBoundingClientRect();
```

Comment:
We can file a separate issue for this, but we should probably put this in the store or something.  `getBoundingClientRect()` is an expensive operation, but we've actually already calculated the page's size in other parts of the `amp-story` codebase.

@Enriqe can you file an issue for this and mark this line with a `TODO`?